How-To-Setup-Internal-Package-Repository.md

```diff
@@ -1,24 +1,6 @@
 # How To Use Package Internalizer To Create Internal Package Source (Business Editions Only)
 
-<!-- TOC -->
-
-- [Summary](#summary)
-- [Organizational Requirements](#organizational-requirements)
-  - [Architecture](#architecture)
-- [Building Your Internal Infrastructure](#building-your-internal-infrastructure)
-  - [Server Pre-Requisites](#server-pre-requisites)
-  - [Internal Package Repositories](#internal-package-repositories)
-    - [Install and Configure Chocolatey Server](#install-and-configure-chocolatey-server)
-  - [Install and Configure Jenkins Server](#install-and-configure-jenkins-server)
-    - [Script: `Get-UpdatedPackage.ps1`](#script-get-updatedpackageps1)
-    - [Script: `Update-ProdRepoFromTest.ps1`](#script-update-prodrepofromtestps1)
-    - [Script: `ConvertTo-ChocoObject.ps1`](#script-convertto-chocoobjectps1)
-    - [Create Jenkins Jobs](#create-jenkins-jobs)
-- [Test the Jenkins Automation (Exercises)](#test-the-jenkins-automation-exercises)
-  - [Submit a new package](#submit-a-new-package)
-  - [Updating a package from the Chocolatey Community Repository](#updating-a-package-from-the-chocolatey-community-repository)
-
-<!-- /TOC -->
+<!-- TOC -->autoauto- [How To Use Package Internalizer To Create Internal Package Source (Business Editions Only)](#how-to-use-package-internalizer-to-create-internal-package-source-business-editions-only)auto    - [Summary](#summary)auto    - [Organizational Requirements](#organizational-requirements)auto        - [Architecture](#architecture)auto    - [Building Your Internal Infrastructure](#building-your-internal-infrastructure)auto        - [Server Pre-Requisites](#server-pre-requisites)auto        - [Internal Package Repositories](#internal-package-repositories)auto            - [Install and Configure Chocolatey Server](#install-and-configure-chocolatey-server)auto        - [Install and Configure Jenkins Server](#install-and-configure-jenkins-server)auto            - [Script: `Get-UpdatedPackage.ps1`](#script-get-updatedpackageps1)auto            - [Script: `Update-ProdRepoFromTest.ps1`](#script-update-prodrepofromtestps1)auto            - [Script: `ConvertTo-ChocoObject.ps1`](#script-convertto-chocoobjectps1)auto            - [Create Jenkins Jobs](#create-jenkins-jobs)auto                - [Jenkins Job Details: Update Test Repository](#jenkins-job-details-update-test-repository)auto                - [Jenkins Job Details: Internalize Package](#jenkins-job-details-internalize-package)auto                - [Jenkins Job Details: Update Production Repository](#jenkins-job-details-update-production-repository)auto    - [Test the Jenkins Automation (Exercises)](#test-the-jenkins-automation-exercises)auto        - [Submit a new package](#submit-a-new-package)auto        - [Updating a package from the Chocolatey Community Repository](#updating-a-package-from-the-chocolatey-community-repository)autoauto<!-- /TOC -->

```

Comment:
@adilio has already mentioned this but I wanted to add it here too. The TOC looks like it messed up.

Reply:
Addressed

Reply:
Still showing the change here as weird

You have AlanWalk's TOC code plugin, which requires you to set an EOL in your code settings, as "auto" is what is being inserted here. https://github.com/AlanWalk/markdown-toc/issues/65

Once you've fixed it and pushed again to your repo, it should update these lines of code.